src/site/content/en/blog/quintoandar/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: How QuintoAndar increased conversion rates and pages per session by improving page performances
+subhead: A project focused on optimizing Core Web Vitals and migrating to Next.js resulted in a 5% increase in conversion rates and a 87% increase in pages per session.
+authors:
+  - jlwagner
```

Comment:
TODO: We need the names of authors (it wasn't evident from the doc), and a 384x384 headshot for each of them, as well as their title for their author data.

cc: @danielasy